src/workerd/api/crypto-impl-asymmetric-test.js

```diff
@@ -0,0 +1,50 @@
+import * as assert from 'node:assert';
+let subtle = self.crypto.subtle;
```

Comment:
`crypto.subtle` should be a global already. There shouldn't be a reason to assign to `subtle` here?